src/components/portfolio/DetailsMainBacktestingList.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import TrendingUpIcon from '@material-ui/icons/TrendingUp';
+import TrendingDownIcon from '@material-ui/icons/TrendingDown';
+import WarningIcon from '@material-ui/icons/Warning';
+import NewReleasesIcon from '@material-ui/icons/NewReleases';
+import SpeedIcon from '@material-ui/icons/Speed';
+import HeightIcon from '@material-ui/icons/Height';
+import { makeStyles } from '@material-ui/core/styles';
+import { useTranslation } from 'react-i18next';
+import StyledNumberFormat from '../shared/StyledNumberFormat';
+import DetailsMainBacktestingListItem from './DetailsMainBacktestingListItem';
+
+const useStyles = makeStyles((theme) => ({
+  list: {
+    width: '100%',
+    flexBasis: '50%',
+  },
+  trendUp: {
+    color: theme.palette.success.light,
+  },
+  trendDown: {
+    color: theme.palette.error.light,
+  },
+  mdd: {
+    color: theme.palette.primary.contrastText,
+  },
+  // TODO replace hard coded
+  beta: {
+    color: '#ffff00',
+  },
+  sharpe: {
+    color: '#ffa500',
+  },
+  cagr: {
+    color: theme.palette.lightBlue.main,
+  },
+}));
+
+type DetailsMainBacktestingListProps = {
+  changeBest: number;
+  changeWorst: number;
+  mddMaxToMin: number;
+  standardDeviation: number;
+  sharpeRatio: number;
+  cagr: number;
+};
+
+const DetailsMainBacktestingList: React.FC<DetailsMainBacktestingListProps> = ({
+  changeBest,
+  changeWorst,
+  mddMaxToMin,
+  standardDeviation,
+  sharpeRatio,
+  cagr,
+}) => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+
+  // TODO replace description placeholder with actual description
+  return (
```

Comment:
I guess that's done.

```suggestion

  return (
```